src/Task/tasks.yml

```diff
@@ -0,0 +1,137 @@
+default:
```

Comment:
Do I understand correctly that this is the default config now that all tasks use? If so then very nice! Could we add comment for the file something like:

    # Default configuration for tasks.

Reply:
Sure